addons/binding/org.openhab.binding.squeezebox/README.md

```diff
@@ -29,7 +29,11 @@ Once it is added as a Thing the Squeeze Server bridge will discover Squeeze Play
 
 ## Binding Configuration
 
-The binding requires no special configuration
+The binding has the following configuration options, which can be set for "binding:squeezebox":
```

Comment:
It would be nice to have a short explanation why the `callbackURL` is needed in some cases (see #3867).

Reply:
Good suggestion. And thanks for the review. I just pushed a commit with the updated documentation.